subprojects/android-test/test-screenshot/src/main/java/com/avito/android/test/screenshot_test/internal/Registry.kt

```diff
@@ -0,0 +1,22 @@
+package com.avito.android.test.screenshot_test.internal
+
+import android.app.Instrumentation
+import android.os.Bundle
+
+class Registry internal constructor() {
+    var instrumentation: Instrumentation? = null
+    var arguments: Bundle? = null
+
+    companion object {
```

Comment:
Don't understand reasons for companion object.
Can we make just `object Registry`?

Reply:
There are no reasons for it) that was old Java code copy paste)